src/oak.py

```diff
@@ -284,6 +287,7 @@ def reconfigure(
         LOGGER.debug(f"Set width attr to {self.width}")
         self.frame_rate = attribute_map["frame_rate"].number_value or DEFAULT_FRAME_RATE
         LOGGER.debug(f"Set frame_rate attr to {self.frame_rate}")
+        device_info = attribute_map["device_info"].string_value or None
```

Comment:
is there a good default for this?

Reply:
Or does it non-deterministically find a camera? if so add a logger warning at creation

Reply:
Yep it will find some camera in an opaque way. Gotchu adding a logger.

None is a good default since we are passing device_info as the device param here which defaults to None: https://github.com/luxonis/depthai/blob/bfa7a48cf03a8a8f5f0d58e5419b6aff3bfcb839/depthai_sdk/src/depthai_sdk/oak_camera.py#L65